### DISCUSSION TITLE
Stop aggregating 'Log::error' with different messages

### DISCUSSION THREAD

**Discussion by maartenderie** [spatie/flareapp.io-roadmap#46] [Ideas]
**Is your feature request related to a problem? Please describe.**
I love the fact that calling `Log::error('Oops')` will should up in Flare. However when I call `Log::error('Doh!')` it will be aggregated in Flare together with the first call. I can get around this by, in Flare app, clicking on the error and clicking on 'messages'.

There are however a couple thing that I would like to do;
- Resolve/snooze different messages separately
- See the count per message on the project-page

** Possible Pitfalls **
If there are a lot of different messages it will become hard to manage. I would be OK with aggregating messages by the first X characters. I personally don't put any variables in the messages (use the context argument), but I can see that not everyone will make that choice. Alternative; aggregate by where the error originated (class name + line number).

**Describe alternatives you've considered**
- Making an Exception-class per thing-that-can-go-wrong; It's just too much work.
- Grepping logs and aggregating them through shell magic; yeah so not using Flare, that's sad

**Additional context**
Example; 'Oh wow 12K+ occurrences!'
![image](https://user-images.githubusercontent.com/19903985/114672449-7350bb00-9d05-11eb-8d32-5209ebb8a93e.png)

Looking closer; 'Ok so it's four (completely) different things'
![image](https://user-images.githubusercontent.com/19903985/114672651-a8f5a400-9d05-11eb-84c8-d562565283d0.png)

**Comment by AlexVanderbist** [comment]
Great suggestion, definitely feels like different log messages should be different errors altogether. Thanks!
